src/javascript/typescript-config.ts

```diff
@@ -47,28 +47,28 @@ export enum TypeScriptModuleResolution {
    *
    * @see https://www.typescriptlang.org/docs/handbook/module-resolution.html#classic
    */
-  CLASSIC = "classic",
+  CLASSIC = "Classic",
 
   /**
    * Resolution strategy which attempts to mimic the Node.js module resolution strategy at runtime.
    *
    * @see https://www.typescriptlang.org/docs/handbook/module-resolution.html#node
    */
-  NODE = "node",
+  NODE = "Node",
```

Comment:
```suggestion
  NODE = "node",
```

Let's keep all of these lower case as per the docs. AFAIK they are case insensitive anyway. 

https://www.typescriptlang.org/tsconfig#moduleResolution
<img width="142" alt="image" src="https://github.com/projen/projen/assets/379814/d4ed6b1d-444f-4519-91dd-521b1163d6dd">

Reply:
I found they look better capitalized properly, but yeah... it makes no difference.